### PR TITLE
Add an appdata XML file

### PR DESCRIPTION
**Pull request by suve**
Solves #8.

### CODE REVIEW

File: pyspacewar.appdata.xml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+		<screenshot>
+			<image>https://mg.pov.lt/pyspacewar/thumb-pyspacewar-0.9.1-game.png</image>
+		</screenshot>
+		<screenshot>
+			<image>https://mg.pov.lt/pyspacewar/thumb-pyspacewar-0.9.1-gravitywars.png</image>
+		</screenshot>
+	</screenshots>
+	
+	<url type="homepage">https://mg.pov.lt/pyspacewar/</url>
+	<url type="bugtracker">https://bugs.launchpad.net/pyspacewar</url>
```
Comment (mgedmin) [inline pyspacewar.appdata.xml]: Actually I'd rather use GitHub issues here:
https://github.com/mgedmin/pyspacewar/issues

File: pyspacewar.appdata.xml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+			<image>https://mg.pov.lt/pyspacewar/thumb-pyspacewar-0.9.1-title.png</image>
+		</screenshot>
+		<screenshot>
+			<image>https://mg.pov.lt/pyspacewar/thumb-pyspacewar-0.9.1-menu.png</image>
+		</screenshot>
+		<screenshot>
+			<image>https://mg.pov.lt/pyspacewar/thumb-pyspacewar-0.9.1-game.png</image>
+		</screenshot>
+		<screenshot>
+			<image>https://mg.pov.lt/pyspacewar/thumb-pyspacewar-0.9.1-gravitywars.png</image>
```
Comment (mgedmin) [inline pyspacewar.appdata.xml]: These are thumbnails.  If you remove the `thumb-` part from the URL, you should get full-size imges.

**Comment by suve** [comment]
Done. Regarding the GitHub Issues vs. Launchpad for bugs - I took the latter from the README, so you might want to edit it.

**Comment by mgedmin** [comment]
> Regarding the GitHub Issues vs. Launchpad for bugs - I took the latter from the README, so you might want to edit it.

Whoops, thank you!

**Pull request merged**